docs/src/priors.md

```diff
@@ -130,13 +130,16 @@ More parameters will be added to this table in the future.
 | EFAC          | TOA uncertainty correction (multiplicative)                                 | `LogNormal[0, 0.25]`                                            |                  |
 | EQUAD         | TOA uncertainty correction (added in quadrature) (μs)                       | `LogUniform[1e-3, 1e+2]`                                        |                  |
 | ECORR         | Correlation between narrowband TOAs measured from the same observation (μs) | `LogUniform[1e-3, 1e+2]`                                        |                  |
+| JUMP          | Instrumental time jump                                                      | `Uniform[-1/(2*F0), 1/(2*F0)]`                                  |                  |
 | KIN           | Orbital inclination (deg)                                                   | `P(KIN) = sin(KIN)`                                             | 0 <= KIN <= 90.  |
 | KOM           | Longitude of ascending node (deg)                                           | `LogNormal[0, 360]`                                             |                  |
 | PHOFF         | Overall phase offset                                                        | `Uniform[-0.5, 0.5]`                                            |                  |
 | PX            | Parallax (mas)                                                              | `P(PX) = 3 / Rmax^3 / PX^4`                                     | 0 <= PX <= Rmax  |
 | SHAPMAX       | Log-scaled inclination parameter                                            | `P(SHAPMAX) = (1 - exp(-SHAPMAX)) / (sqrt(2 * exp(SHAPMAX) - 1))`| 0 <= SHAPMAX    |
 | SINI          | Orbital sine-inclination                                                    | `P(SINI) = SINI / sqrt(1 - SINI^2)`                             | 0 <= SINI <= 1   |
 | STIGMA        | Orthometric Shapiro delay parameter                                         | `P(STIGMA) = 4 * STIGMA / (1 + STIGMA^2)^2`                     | 0 <= STIGMA <= 1 |
+| T0            | Epoch of periapsis passage                                                  | `Uniform[T0_1 - PB/2, T0_1 + PB/2]`                             | T0_1 is the default value of T0. |
```

Comment:
I think these might be a little too broad, looking at some results.  We want to make sure that the solution stays near the starting point and doesn't jump to near an alternate.  So maybe +/-PB/4 or even smaller?

Reply:
I think this should be the default since it is the most uninformative choice. If more restrictive priors are needed those can be specified in the prior file.